rpms/build/scripts/wheel.sh

```diff
@@ -26,19 +26,25 @@ pip3.4 install ipython==5.1.0 /archives/ipykernel.zip
 pip3.4 wheel ipython==5.1.0 /archives/ipykernel.zip
 rm -f decorator-4.2.1-py2.py3-none-any.whl \
    google_auth-1.4.0-py2.py3-none-any.whl \
+   ipykernel-4.7.0.dev0-py3-none-any.whl \
    ipykernel-4.8.0-py3-none-any.whl \
    ipykernel-4.8.1-py3-none-any.whl \
    ipython_genutils-0.2.0-py2.py3-none-any.whl \
    jupyter_client-5.2.2-py2.py3-none-any.whl \
+   jupyter_client-4.4.0-py2.py3-none-any.whl \
    jupyter_core-4.4.0-py2.py3-none-any.whl \
+   pexpect-4.2.1-py2.py3-none-any.whl \
    pexpect-4.3.1-py2.py3-none-any.whl \
    prompt_toolkit-1.0.15-py3-none-any.whl \
    ptyprocess-0.5.2-py2.py3-none-any.whl \
    Pygments-2.2.0-py2.py3-none-any.whl \
    python_dateutil-2.6.0-py2.py3-none-any.whl \
+   python_dateutil-2.6.1-py2.py3-none-any.whl \
+   pyzmq-16.0.2-cp34-cp34m-manylinux1_x86_64.whl \
    pyzmq-16.0.4-cp34-cp34m-manylinux1_x86_64.whl \
    setuptools-18.5-py2.py3-none-any.whl \
+   simplegeneric-0.8.1-py3-none-any.whl \
    six-1.11.0-py2.py3-none-any.whl \
-   tornado-4.5.3-cp34-cp34m-linux_x86_64.whl \
+   tornado-5.0-cp34-cp34m-linux_x86_64.whl \
    traitlets-4.3.2-py2.py3-none-any.whl
```

Comment:
The methodology that I have been using for producing this file is to build a container with the desired dependency and check which versions are reported by `pip freeze`.  I do not know whether this result matches that or whether it matters in any significant why if it does not.